testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -155,15 +155,15 @@ def test_rmsd_matrix_with_superimposition_custom_weights(self, ens1):
         conf_dist_matrix = encore.confdistmatrix.conformational_distance_matrix(
             ens1,
             encore.confdistmatrix.set_rmsd_matrix_elements,
-            select="name CA",
+            selection="name CA",
```

Comment:
As already pointed out by @lilyminium, `selection` has been changed to `select` in #2528